mods/persistence/game/area/area_space.dm

```diff
@@ -0,0 +1,2 @@
+/area/space
```

Comment:
You might wanna just clear the flags you don't want in Initialize() since if more flags are added in the base code this could get screwy. Just an observation though.